modules/aruco/tutorials/aruco_detection/aruco_detection.markdown

```diff
@@ -518,74 +533,88 @@ this filtering process.
 
 It must be noted that in most cases it is a question of balance between detection capacity
 and performance. All the considered contours will be processed in the following stages, which usually have
-a higher computational cost. So, it is preferred to discard wrong candidates in this stage than in the later stages.
+a higher computational cost. So, it is preferred to discard invalid candidates in this stage than in the later stages.
 
 On the other hand, if the filtering conditions are too strict, the real marker contours could be discarded and,
 hence, not detected.
 
-- ```double minMarkerPerimeterRate```, ```double maxMarkerPerimeterRate```
+- `double minMarkerPerimeterRate`
+- `double maxMarkerPerimeterRate`
 
-These parameters determine the minimum and maximum size of a marker, concretely the maximum and
-minimum marker perimeter. They are not specified in absolute pixels values, instead they are
+These parameters determine the minimum and maximum size of a marker, specifically the minimum and maximum marker perimeter. They are not specified in absolute pixel values, instead they are
 specified relative to the maximum dimension of the input image.
 
 For instance, a image with size 640x480 and a minimum relative marker perimeter of 0.05 will lead
 to a minimum marker perimeter of 640x0.05 = 32 pixels, since 640 is the maximum dimension of the
-image. The same applies for the ```maxMarkerPerimeterRate``` parameter.
+image. The same applies for the `maxMarkerPerimeterRate` parameter.
 
-If the ```minMarkerPerimeterRate``` is too low, it can penalize considerably the detection performance since
+If the `minMarkerPerimeterRate` is too low, it can penalize considerably the detection performance since
 many more contours would be considered for future stages.
-This penalization is not so noticeable for the ```maxMarkerPerimeterRate``` parameter, since there are
+This penalization is not so noticeable for the `maxMarkerPerimeterRate` parameter, since there are
 usually many more small contours than big contours.
-A ```minMarkerPerimeterRate``` value of 0 and a ```maxMarkerPerimeterRate``` value of 4 (or more) will be
+A `minMarkerPerimeterRate` value of 0 and a `maxMarkerPerimeterRate` value of 4 (or more) will be
 equivalent to consider all the contours in the image, however this is not recommended for
-the performance reasons.
+performance reasons.
 
 Default values:
 
- ```minMarkerPerimeterRate``` : 0.03, ```maxMarkerPerimeterRate``` : 4.0
-
+- `minMarkerPerimeterRate`: 0.03
+- `maxMarkerPerimeterRate`: 4.0
 
-- ```double polygonalApproxAccuracyRate```
+---
+- `double polygonalApproxAccuracyRate`
```

Comment:
The same here and other places.

Reply:
I removed the code attribute from the headings that introduce the parameters. With the code attribute, the headings didn't stand out. Instead, I made them plain text and changed the default value examples to include the parameter's type and to be a complete assignment statement instead of a colon followed by a value in plain text. I think this is visually clearer than the horizontal line or having code as headers.